mlflow/tracking/fluent.py

```diff
@@ -233,7 +233,7 @@ def start_run(
             environment variable, ``MLFLOW_EXPERIMENT_ID`` environment variable,
             or the default experiment as defined by the tracking server.
         run_name: Name of new run. Used only when ``run_id`` is unspecified. If a new run is
-            created and ``run_name`` is not specified, a unique name will be generated for the run.
+            created and ``run_name`` is not specified, a random name will be generated for the run.
```

Comment:
unique name should be a good representation. Our goal is to create a run name that doesn't conflict with other run names, the "random generator" is just the approach to achieve the goal.

Reply:
Yes, however there is still chance that it collides. The reason why I checked uniqueness is that, I want to use the name (instead of uuid) as the *unique* identifier to be referred by my other parts of system (e.g. Obsidian double link). 

IMHO when reading the word "unique", I do expect it to be really unique, such as when we say "primary key in database is unique".

Reply:
@fzyzcjy yeah random seems more correct here.

Reply:
> What about e.g. "a random name (usually unique)"

I'd use this if run names had to be unique.